pkg/reconciler/pubsub/topic/topic.go

```diff
@@ -124,7 +124,7 @@ func (r *Reconciler) reconcileTopic(ctx context.Context, topic *v1alpha1.Topic)
 	}
 	defer client.Close()
 
-	t := client.Topic(topic.Status.ProjectID)
+	t := client.Topic(topic.Status.TopicID)
```

Comment:
```suggestion
	t := client.Topic(topic.Spec.TopicID)
```

I think.

Reply:
that's right, had the wrong fix...
Thanks @Harwayne 
giving it another try now